### PR TITLE
docs: add docker version requirement in demo/quickstart

### LINKED ISSUE #956: segmentation fault when trying demo under the standalone mode
image
A segmentation fault occurs during the operation of Demo with The Standalone Mode ../openmldb/bin/openmldb --host 127.0.0.1 --port 6527, which may be caused by insufficient memory of the machine.

### PR DESCRIPTION
**Pull request by zhanghaohit**
fix #956

### CODE REVIEW

**Review by lumianph — APPROVED** [review]
LGTM

File: demo/README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ w2 as (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
 
 ## 2. Demo with The Cluster Mode
 
+> :warning: Required docker version >= 18.03
```
Comment (aceforeverd) [inline demo/README.md]: it's docker desktop for macOS intel ?
not docker package for linux

Comment (aceforeverd) [reply demo/README.md]: it's the docker-engine/moby's version.

Comment (zhanghaohit) [reply demo/README.md]: docker package you mean docker client? or docker-ce?

Comment (aceforeverd) [reply demo/README.md]: whichever contains docker engine. that's fine

**Review by aceforeverd — APPROVED** [review]

**Comment by lumianph** [comment]
[bugfix tracking](https://trello.com/c/wEQ9HW0k/25-bugfix-tracking)

**Comment by zhanghaohit** [comment]
[docs reorganize, review and udpate](https://trello.com/c/EW0F7yOb/1-docs-reorganize-review-and-udpate)

**Pull request merged**